core/src/test/java/org/bitcoinj/utils/BlockFileLoaderTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 Tim Straser.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.utils;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.List;
+
+public class BlockFileLoaderTest {
+
+    private final File TEST_DIRECTORY = new File("./test");
```

Comment:
I think we should create temp directories in `/tmp` (or similar on Windows/MacOS). Or alternatively in the `build` directory.

In any case, the directory name should be randomized, as `test` is much too likely to clash with existing directories. Something similar to what `File.createTempFile()` does.